### PR TITLE
Speed up integration-tests

### PR DESCRIPTION
**Pull request by sainoe**
* Use muti-stage builds in the integration-tests [Dockerfile](https://github.com/cosmos/interchain-security/blob/sainoe/integration-test-speedup/Dockerfile) to remove the installation and compilation time of Go, Rust and Hermes.

* Speed up the integration-tests execution time from ~25 to ~10 mins in Github checks .

### CODE REVIEW

**Review by mpoke — APPROVED** [review]
Nice 🚀

**Review by smarshall-spitzbart — APPROVED** [review]
Nice!

**Comment by jtremback** [comment]
Nice

File: integration-tests/main.go [hunk 1]
```diff
@@ -21,7 +23,7 @@ func main() {
 		s.runStep(step, verbose)
 	}
 
-	println("test successful")
+	fmt.Printf("test successful - time elapsed %v\n", time.Since(start))
```
Comment (jtremback) [inline integration-tests/main.go:26]: Nice flex 💪

**Pull request merged**